### DISCUSSION TITLE
[✨] Can we have a flag to remove the "performance loss" and "waterfall request" notices in the console?

### DISCUSSION THREAD

**Discussion by jazoom** [QwikDev/qwik-evolution#119] [Proposals For Qwik]
### Is your feature request related to a problem?

They are very attention/focus-grabbing and I really don't need to see it every time I start up the server.

Also, it means I have to have my terminal window taking up more room at the bottom of VS Code, or I need to scroll up to see the URL to click on.

As an aside, when I first tried Qwik City a few days ago, I thought this was some kind of error. From looking around the internet it seems I wasn't the only one who thought that. The colour of the text and the exclamation marks mark it seem like they're errors.

### Describe the solution you'd like

Option 1: Remove them entirely and just make it clear in the docs. I presume this won't be acceptable since you clearly want it in everyone's face, presumably to prevent unnecessary bug reports on performance.

Option 2: A simple CLI option to disable these notices.

### Describe alternatives you've considered

I'd be happy with either option.

**Comment by gioboa** [comment]
The messages are logged [here](https://github.com/QwikDev/qwik/blob/b35665d677022fc8c33c7959c41a4f185590963e/packages/qwik/src/optimizer/src/plugins/vite-dev-server.ts#L244)

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
